frontend/src/models/userEntity.ts

```diff
@@ -0,0 +1,7 @@
+export interface UserEntity {
```

Comment:
Entity is for spring database.
```suggestion
export interface User {
```